app/controllers/projects/edit/steps/rezoning.js

```diff
@@ -9,6 +9,13 @@ export default class RezoningController extends Controller {
     return false;
   }
 
+  @computed('hasAnsweredAll', 'model.needUnderlyingZoning', 'model.needCommercialOverlay', 'model.needSpecialDistrict')
+  get hasAnsweredLogically() {
+    if (this.get('hasAnsweredAll') && (this.get('model.needUnderlyingZoning') || this.get('model.needCommercialOverlay') || this.get('model.needSpecialDistrict'))) return true;
+    if (this.get('hasAnsweredAll')) return false;
+    return null;
+  }
```

Comment:
I think most of this should be moved to the Project model, including `hasAnsweredAll`. Notice how all the dependent keys for these computed properties reference some part of the model? Since they're all downstream of that model, I think it makes more sense to include them there. Thoughts?

The tricky part is the action, see below...